arroyo/processing/strategies/run_task.py

```diff
@@ -14,6 +14,42 @@
 TResult = TypeVar("TResult")
 
 
+class RunTask(ProcessingStrategy[TPayload]):
+    """
+    Basic strategy to run a custom processing function on a message.
+    """
+
+    def __init__(
+        self,
+        function: Callable[[Message[TPayload]], TResult],
+        next_step: ProcessingStrategy[TResult],
+    ) -> None:
+        self.__function = function
+        self.__next_step = next_step
+
+    def submit(self, message: Message[TPayload]) -> None:
+        self.__next_step.submit(
+            Message(
+                message.partition,
+                message.offset,
+                self.__function(message),
+                message.timestamp,
+            )
+        )
+
+    def poll(self) -> None:
+        self.__next_step.poll()
+
+    def join(self, timeout: Optional[float] = None) -> None:
+        self.__next_step.join(timeout)
+
+    def close(self) -> None:
+        self.__next_step.close()
```

Comment:
This is the only functional change in this PR. We are now calling next_step.close() in the close method instead of the join method as transform did previously.